apps/ideas/views.py

```diff
@@ -79,22 +71,23 @@ class IdeaListView(AbstractIdeaListView, DisplayProjectOrModuleMixin):
     filter_set = IdeaFilterSet
 
 
-class AbstractIdeaDetailView(ProjectMixin,
-                             rules_mixins.PermissionRequiredMixin,
-                             CanonicalURLDetailView):
+class AbstractIdeaDetailView(
+    ProjectMixin, rules_mixins.PermissionRequiredMixin, CanonicalURLDetailView
+):
     get_context_from_object = True
 
 
 class IdeaDetailView(AbstractIdeaDetailView):
     model = models.Idea
-    queryset = models.Idea.objects.annotate_positive_rating_count()\
-        .annotate_negative_rating_count()
-    permission_required = 'a4_candy_ideas.view_idea'
+    queryset = (
+        models.Idea.objects.annotate_positive_rating_count().annotate_negative_rating_count()
+    )
```

Comment:
This line is for example longer than 88, but tbh I dont know how to change that (just moving a part of it to the next line is still considered to be one line in flake), unless writing a function annotate_rating_counts() :) Do you have an idea?

Reply:
maybe 
```
queryset = models.Idea.objects.all()
queryset.annotate_positive_rating_count()
queryset.annotate_negative_rating_count()
```

Reply:
That doesnt work as it is just the defintion of the class attributes.. I can maybe try and put it into innit? Will do!